packages/blockchain-wallet-v4-frontend/src/scenes/Exchange/GetStarted/StatusBar/Statuses/__snapshots__/Rejected.spec.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Rejected renders correctly 1`] = `<Rejected__Wrapper />`;
```

Comment:
Is it the correct snapshot?

Reply:
That's a funny one indeed, gonna check that

Reply:
Found